LiteCore/Support/FilePath.cc

```diff
@@ -426,6 +426,17 @@ namespace litecore {
         return s.st_size;
     }
 
+    time_t FilePath::lastModified() const {
+        struct stat s;
+        if (stat_u8(path().c_str(), &s) != 0) {
+            if (errno == ENOENT)
+                return -1;
```

Comment:
Seems weird to me that ENOENT isn't considered an error here. Is it useful to have this call succeed if the file is missing?

Reply:
My basis for this was on the `dataSize` method